datalad_container/containers_run.py

```diff
@@ -131,14 +132,41 @@ def __call__(cmd, container_name=None, dataset=None,
             # just prepend and pray
             cmd = container['path'] + ' ' + cmd
 
+        extra_inputs_ = container.get("extra-input",[])
```

Comment:
we aren't really flake'ing etc , but we do love to separate them out:

```suggestion
        extra_inputs_ = container.get("extra-input", [])
```